src/documentation/ntfs_to_netex_france_specs.md

```diff
@@ -140,3 +162,47 @@ warnings:
 - in monomodalStopPlace: all stop_points should have the same name. If not, 2 StopPlace must be created,
 - multimodalStopPlace: show the higher rated mode value only
 
+## lignes.xml
+
+### Top level structure
+
+Example: 
+
+>   <?xml version="1.0" encoding="UTF-8"?>
+    <CompositeFrame 
+            id="FR:CompositeFrame:NETEX-20190709T093214Z:LOC" 
+            version="1.8">
```

Comment:
Probably another version?
```suggestion
            version="any">
```

Reply:
We'll leave it as it is for now. Following what we have in NeTEx IDF at the moment.